custom-generator/src/main/java/io/serverlessworkflow/generator/AllAnyOneOfSchemaRule.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2020-Present The Serverless Workflow Specification Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.serverlessworkflow.generator;
+
+import static org.apache.commons.lang3.StringUtils.*;
```

Comment:
nit; style-wise we normally don't do star imports. I think the fmt plugin you have there can fix this automatically.

Reply:
I forgot I have to remove StringUtils dependency